### ISSUE TITLE
Brightness Question!

### ISSUE THREAD

**Issue by 6707110211** [5T33Z0/Lenovo-T530-Hackintosh-OpenCore#45]
Hello to start off I want to say thankyou so much for your guide! I would never be able to fix sleep withought your binary patching guide! I have a t431s thinkpad and my brightness at boot is always around 75% of the actual brightness even when it says its full. But when I close the lid and sleeps (thx to you :) and I open it the brightness actually goes to 100%. I was wondering if this crossed with the brightness issue on your end as well. When I get home im going to try to use your boot-arg to see if it fixes anything!

**Comment by 5T33Z0** [comment]
This happened to me when using the SSDT-PNLF generated by SSDTtime. Since then, I've switched to the SSDT-PNLF Sample contained in the OpenCore Package. Read the comments in the .dsl file for modifying the template.

**Comment by 6707110211** [comment]
Thankyou so much!!!!! Do you have any donations???

**Comment by 5T33Z0** [comment]
No, but if you like Synthwave music you could buy my music on bandcamp if you like it: https://stansteez.bandcamp.com/music
